### PR TITLE
Update variables.js

### PR DESCRIPTION
**Pull request by Tagurooo**
add Taguroo website link

### CODE REVIEW

File: utils/variables.js [hunk 1]
```diff
@@ -714,6 +714,16 @@ const IT_3101 = [
         code: "",
       },
     },
+  },
+    {
```
Comment (owenfalculan) [inline utils/variables.js:718]: remove extra {

**Review by owenfalculan — CHANGES_REQUESTED** [review]

File: utils/variables.js (hunk 1 above)
Comment (owenfalculan) [reply utils/variables.js:718]: remove that in your local then type

git add .
git commit -m "update changes"
git push

**Comment by Tagurooo** [comment]
Sir na edit kona po, pa approved po hehehe

**Comment by owenfalculan** [comment]
paclose

**Comment by Tagurooo** [comment]
Updated.

**Pull request closed without merging**